### PR TITLE
feat(@schematics/angular): update to codelyzer 5

### PR DESCRIPTION
**Pull request by mgechev**
This PR also updates the hello world sample app and migrates its `tslint.json` file.

### CODE REVIEW

File: packages/angular_devkit/build_angular/package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "@angular/platform-server": "^8.0.0-beta.9",
     "@angular/router": "^8.0.0-beta.9",
     "@angular/service-worker": "^8.0.0-beta.9",
-    "codelyzer": "^4.2.1",
+    "codelyzer": "^5.0.0",
```
Comment (mgechev) [inline packages/angular_devkit/build_angular/package.json]: @clydin I didn't find the corresponding tslint file here. Let me know if there are other migrations necessary.

File: tests/angular_devkit/build_angular/hello-world-app/tslint.json [hunk 2]
```diff
@@ -130,12 +130,12 @@
       "kebab-case"
     ],
     "no-output-on-prefix": true,
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "no-inputs-metadata-property": true,
```
Comment (alan-agius4) [inline tests/angular_devkit/build_angular/hello-world-app/tslint.json]: There seems to be more rules to be updated see https://github.com/angular/angular-cli/pull/14007/files

File: packages/angular_devkit/build_angular/package.json (hunk 1 above)
Comment (clydin) [reply packages/angular_devkit/build_angular/package.json]: That is there for dev/testing purposes only.  No need for further action.

**Comment by alan-agius4** [comment]
LGTM, other than you need to update the `yarn.lock`

**Pull request merged**